advanced_functionality/causal-inference/causal-inference-container.ipynb

```diff
@@ -0,0 +1,588 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.        method="fixed", numeric_split_points=[1.1, 1.4]
</pre><p>As I asked in another file - please explain how these discretization ranges were selected. How can customers choose their own ranges if they have different datasets?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3130/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>